### PR TITLE
Ks8500 runner update1

### PR DESCRIPTION
**Pull request by TorstenIversen**
Updated to latest stable OpenTap
Added python dependencies
Increased memory default to make room for python based post processing

### CODE REVIEW

**Comment by TorstenIversen** [comment]
Please hold the merge, we expect to do a minor update to the runner version number in Dockerfile before releasing

**Comment by TorstenIversen** [comment]
Update to 1.11.4 is done and tested, we should be able to proceed with review

File: ks8500_runner/changelog.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## [unreleased]
```
Comment (jmaciasportela) [inline ks8500_runner/changelog.md:3]: You need to define a version number to be tracked on the global version.

**Review by jmaciasportela — CHANGES_REQUESTED** [review]

File: ks8500_runner/changelog.md (hunk 1 above)
Comment (TorstenIversen) [reply ks8500_runner/changelog.md]: Not sure exactly what is allowed for versioning of a component. I added a 4 digit number to allow tracking the wrapped runner (first 3 digits) and changes to the Open Nebula Component (last digit). Let me know if this is not acceptable

**Review by jmaciasportela — APPROVED** [review]

**Pull request merged**